types/hot-formula-parser/index.d.mts

```diff
@@ -0,0 +1,3 @@
+import p = require('hot-formula-parser');
+
+export = p;
```

Comment:
I think you shouldn't use `export = ...` in an `.mts`/`.d.mts` file. It's a TypeScript error to do this if `module` is set to `node16` or higher.
Please use the ESM export syntax instead.